### DISCUSSION TITLE
🧩 FEATURE REQUEST:  GitHub Related Reputation

### DISCUSSION THREAD

**Discussion by RatulMaharaj** [dailydotdev/daily#1566] [Feature Requests]
### Concise Description

It would be really nice if we could earn reputation via GitHub

### Description

I think it would be really cool if we could earn or be rewarded _reputation_ via github contributions e.g. assisting in solving open issues, or making code contributions.

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @RatulMaharaj,

That's a great idea. Let me think about it

**Comment by sabdailydev** [comment]
Hi @RatulMaharaj , thanks for the suggestion. Were you referring to contributions to daily.dev or in general across github?

**Comment by RatulMaharaj** [comment]
Hi @sabdailydev 

I was actually referring to contributions made to the daily.dev repos.

Although, all of GitHub could be a really interesting idea as well.

**Comment by theshekslaw** [comment]
Hi @RatulMaharaj , thanks for the suggestion.
